src/twist_marker.cpp

```diff
@@ -86,6 +87,19 @@ class TwistMarker
     }
   }
 
+  void update(const geometry_msgs::msg::TwistStamped & twist)
+  {
+    using std::abs;
+
+    marker_.points[1].x = twist.twist.linear.x;
+
+    if (abs(twist.twist.linear.y) > abs(twist.twist.angular.z)) {
+      marker_.points[1].y = twist.twist.linear.y;
+    } else {
+      marker_.points[1].y = twist.twist.angular.z;
+    }
```

Comment:
It is generally frowned upon to copy&paste code instead of reusing it.

```suggestion
    update(twist.twist);
```